### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by hinata1031** [tcgoetz/GarminDB#53]
I'm a beginner.
I couldn't solve this error.Please tell me how to sovle it.
  File "PyInstaller/loader/rthooks/pyi_rth__tkinter.py", line 30, in <module>
FileNotFoundError: Tcl data directory "/var/folders/74/f7zsky854_z84s16nsrpb1gc0000gn/T/_MEIGv6D1D/tcl" not found.

**Comment by tcgoetz** [comment]
I need some more context to interpret the error. Can you attach the complete screen output and garmin.log?

**Comment by hinata1031** [comment]
Updating existing DBs by downloading from Garmin Connect
Traceback (most recent call last):
  File "PyInstaller/loader/rthooks/pyi_rth__tkinter.py", line 30, in <module>
FileNotFoundError: Tcl data directory "/var/folders/74/f7zsky854_z84s16nsrpb1gc0000gn/T/_MEIGv6D1D/tcl" not found.
[44593] Failed to execute script pyi_rth__tkinter

this is all

**Comment by tcgoetz** [comment]
what about garmin.log?

**Comment by tcgoetz** [comment]
Not enough information to debug this issue.

**Comment by durandom** [comment]
@tcgoetz same here. Happens with the binary release.

```
❯ ./garmin -v
Traceback (most recent call last):
  File "PyInstaller/loader/rthooks/pyi_rth__tkinter.py", line 30, in <module>
FileNotFoundError: Tcl data directory "/var/folders/dr/pqhvw6m57cz_p9vfb7v_9rhc0000gn/T/_MEIRmouy2/tcl" not found.
[62659] Failed to execute script pyi_rth__tkinter
```

there's no `garmin.log`

**Comment by tcgoetz** [comment]
@durandom Looks like a pyinstaller issue. Looks like it happens even before garmin.py is run.

What OS version? Do previous binary releases work for you? Does the source release work?

**Comment by durandom** [comment]
I'm on MacOS 10.15.3 (19D76)

Just tried the 1.3 release, same:

```
 ~/Downloads/GarminDb_Darwin_1.3.0 |                                                                                                                                                                                       mhild
❯ ./garmin -v
Traceback (most recent call last):
  File "PyInstaller/loader/rthooks/pyi_rth__tkinter.py", line 30, in <module>
FileNotFoundError: Tcl data directory "/var/folders/dr/pqhvw6m57cz_p9vfb7v_9rhc0000gn/T/_MEIgRWoVG/tcl" not found.
[86812] Failed to execute script pyi_rth__tkinter
```

**Comment by durandom** [comment]
Oh, and the source release works for me....

**Comment by tcgoetz** [comment]
Pyinstaller issue: https://github.com/pyinstaller/pyinstaller/issues/3753
setuptools issue: https://github.com/pypa/setuptools/issues/1963

Need to merge fixes to master and spin a new release.

**Comment by tcgoetz** [comment]
Fixed in v1.5.0. Please test and validate.

**Comment by durandom** [comment]
1.5.0 works for me. thnks
